vars/stepResult.groovy

```diff
@@ -114,7 +114,7 @@ def call(Map config= [:]) {
                  ".  " + comment_url
 
       if (commitPragma(pragma: "Skip-PR-comments") == "true") {
-          emailext subject: "${JOB_NAME} ${config.name} stage ${config.result}",
+          emailext subject: "${config.name} status: ${config.result}",
```

Comment:
This doesn't seem to align with the PR description.  Isn't this removing the job name from the email subject?

Reply:
Ick.  Yeah.  Let me take another look at this.  It must just be the one other part that needs to be in this PR.